src/features/components/templates/navigation/main-nav/content/resources/data.ts

```diff
@@ -38,6 +38,13 @@ export const resourcesNavSectionContent: SmartNavContent[] = [
     },
     {
         id: 5,
+        data: {
+            title: '_t_Acuity_t_',
+            url: { type: 'internal', to: '/acuity/' },
+        },
+    },
+    {
+        id: 6,
         data: {
             title: '_t_Status page_t_',
             url: { type: 'non-company', href: 'https://deriv.statuspage.io', show_eu_modal: true },
```

Comment:
`target="_blank"` is needed here

Reply:
hi @hasan-deriv  since its not my change I don't want to change here.
and I saw there are several places missing target ='_blank' better we add all at once.